app_dart/lib/src/service/github_checks_service.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:cocoon_service/src/foundation/github_util.dart';
+import 'package:cocoon_service/src/model/github/checks.dart';
+import 'package:cocoon_service/src/model/luci/buildbucket.dart';
+import 'package:github/github.dart' as github;
+
+import '../../cocoon_service.dart';
+import '../model/luci/push_message.dart' as push_message;
+import 'luci_build_service.dart';
+
+class GithubChecksService {
+  GithubChecksService(this.config, {GithubChecksUtil githubChecksUtil})
+      : githubChecksUtil = githubChecksUtil ?? const GithubChecksUtil();
+
+  Config config;
+  GithubChecksUtil githubChecksUtil;
+
+  Future<void> handleCheckSuite(
```

Comment:
Nit: add url to the API documentation in comments. Also apply to other functions.

Reply:
Done